hazelcast/src/test/java/com/hazelcast/test/annotation/RunParallel.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.hazelcast.test.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ *  Annotation to mark tests to be run in parallel. Used with {@link com.hazelcast.test.HazelcastTestRunner}
+ */
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface RunParallel {
```

Comment:
You can't just run a unit test with two runners. For parameterized tests, you need to use HazelcastTestRunner, and HazelcastTestRunner internally uses either parallel or serial runner. But we need to give that information to the HazelcastTestRunner which is why I introduced the RunParallel annotation and made HazelcastTestRunner to use serial runner by default. If the test is marked with RunParallel, HazelcastTestRunner will use parallel runner.